Core/src/main/java/com/plotsquared/core/plot/flag/implementations/PlotTitleFlag.java

```diff
@@ -56,16 +62,17 @@ public PlotTitleFlag parse(@NonNull String input) throws FlagParseException {
         if (!input.contains("\"")) {
             return new PlotTitleFlag(new PlotTitle(input, ""));
         }
-        input = input.substring(input.indexOf("\""));
-        input = input.substring(0, input.lastIndexOf("\"") + 1);
-        String[] inputs = input.split("\"");
+
+        var split = StringMan.splitMessage(input);
+
+        if(split.isEmpty() || split.size() > 2){
```

Comment:
Please format this to match our code style :p

Reply:
reformatted